Note to avoid confusion with the example (#792)
Qiskit/qiskit-nature: README.md

using the VQE (Variational Quantum Eigensolver) algorithm to compute
the ground-state (minimum) energy of a molecule.

> > Note: The following example belongs to the *main* branch (version >= 0.5).
> > However, new users will commonly have the *stable* version installed via pip.
> > The working example for the *stable* branch can be found at [its README.md file](https://github.com/Qiskit/qiskit-nature/blob/stable/0.4/README.md) or at the project description on [Pypi](https://pypi.org/project/qiskit-nature/).

Reviewer: I would not refer to "main" or "stable" to avoid outdated data after the release. A more atemporal reference would avoid that. 

```suggestion
> Note: The following example belongs to the version >= 0.5 .
> The working example for version <0.5 can be found at [old README.md file](https://github.com/Qiskit/qiskit-nature/blob/stable/0.4/README.md).
```
